gflownet/envs/amp.py

```diff
@@ -55,52 +79,36 @@ def __init__(
         self,
         max_seq_length=50,
         min_seq_length=1,
+        # Not required in env. But used in config_env in MLP. TODO: Find a way out
         n_alphabet=20,
```

Comment:
Since `AMINO_ACIDS` is used instead to determine the alphabet and there is no alternatives implemented, this should not be an argument of the environment.

Reply:
Yeah I agree. I need to change the activelearning MLP proxy implementation so that it's compatible with the AMP environment without `n_alphabet`. I was planning to create something similar to `policy_fixed_input` but that would depend on the `proxy_state_format` -- it is a slightly involved change so I was planning to do it in a separate PR